src/shared/bootspec.c

```diff
@@ -95,34 +95,37 @@ int boot_entry_load(const char *path, BootEntry *entry) {
                 if (IN_SET(*strstrip(buf), '#', '\0'))
                         continue;
 
-                p = strchr(buf, ' ');
-                if (!p) {
+                p = buf;
+                r = extract_first_word(&p, &field, NULL, 0);
```

Comment:
`line_get_key_value` in boot.c uses `" \t"`. Please specify the same here to override the default. It shouldn't matter too much, but there's always the possibility that somebody inserts '\r' in the file, and then we want bootctl and sd-boot to treat them the same.

Maybe also add a comment mentioning that this code must match what boot.c does?

Reply:
Thank you for the comment, and sorry to late reply. Updated.